common/djangoapps/cme_registration/tests/registration_tests.py

```diff
@@ -53,7 +53,6 @@ def setUp(self):
                           'specialty': 'specialty',
                           'sub_specialty': 'sub_specialty',
                           'affiliation': 'affiliation',
-                          'other_affiliation': 'other_affiliation',
```

Comment:
Since this value is always set to `None` during real use, shouldn't we set it to `None` here?